mock/ext-api-dyson/get/cosmos-api-commands4.js

```diff
@@ -37,8 +37,7 @@ module.exports = {
         }
 
         // fallback
```

Comment:
do we need comment //fallback here?

Reply:
Negligible value, but does it hurt?

Reply:
what's the purpose of comment?

Reply:
Helps me understand why the Not implemented there when I revisit this code in 3 months and already forgot part of it.  It hints at the code structure of some combinations covered through some if/switch cases, and everything else falls through till the fallback.

Reply:
`return {error: "Not implemented"};` this is self explanatory. you can always go to history to see changes. I would avoid adding anything that's not needed